index.html

```diff
@@ -4,19 +4,19 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=0.5">
     <title>React Highcharts</title>
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
```

Comment:
Shit, to be honest - I always was thinking that "./" and "/" - it is just the same... )))) Ahah, I am so stupid )))

Reply:
To be honest, I still confused with the relative and the absolute path. And sometimes I need to read documentation :)